moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -1908,9 +1909,11 @@ srdf::Model::Group* MoveItConfigData::findGroupByName(const std::string& name)
 
   // Check if subgroup was found
   if (searched_group == nullptr)  // not found
-    ROS_FATAL_STREAM("An internal error has occured while searching for groups. Group '" << name
-                                                                                         << "' was not found "
-                                                                                            "in the SRDF.");
+  {
+    RCLCPP_ERROR_STREAM(LOGGER, "An internal error has occurred while searching for groups. Group '"
+                                    << name << "' was not found  in the SRDF.");
+    exit(-1);
```

Comment:
should this throw instead of calling exit?

Reply:
I could just remove it, to be a straight port.  This was written by Dlu, I guess the thinking, its better to exit here then carry on?

Reply:
```suggestion
    throw std::runtime_error(name + " was not found in the SRDF");
```